homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -127,6 +128,13 @@
     }
 )
 
+SERVICE_SCHEMA_GOTO = VACUUM_SERVICE_SCHEMA.extend(
+    {
+        vol.Required("x_coord"): vol.Coerce(int),
```

Comment:
It'd probably make sense to use voluptuous' `Range` to define the minimum and maximum value for these?

Reply:
Actually I don't know the limits. The map seems to be centered around (25000, 25000) as suggested [here](https://community.home-assistant.io/t/howto-xiaomi-vacuum-zoned-cleaning/51293). The maximum/minimum values from there depends on how big the map is I guess. Also, it does accept negative values it's not like the coordinates are required to be positive either